vertx-web-graphql/vertx-web-graphiql/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "vertx-web-graphiql",
+  "version": "0.1.0",
+  "private": true,
+  "homepage": ".",
+  "dependencies": {
+    "graphiql": "^0.13.2",
+    "graphql": "^14.4.2",
+    "isomorphic-fetch": "^2.2.1",
+    "lodash": "^4.17.15",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-scripts": "3.0.1"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
```

Comment:
follow up from the travis file, here you could do: `react-scripts build && cp -Rf build/* ../src/main/resources/io...`

Reply:
But then the npm build will only work on *nix like systems.

Reply:
I just realized we're talking about Travis only... And given the GraphiQL app is for dev only, the additional files will not be a problem.

Reply:
I also just realized that adding all files makes the vertx-web JAR go from less than 500KB to more than 1MB. So I'll stick to copying only the required files.

Reply:
I also just realized that adding all files makes the vertx-web JAR go from less than 500KB to more than 1MB :smile: So I'll stick to copying only the required files.